org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/format/DOMAttributeFormatter.java

```diff
@@ -143,4 +164,12 @@ private char getQuotationAsChar() {
 	private EnforceQuoteStyle getEnforceQuoteStyle() {
 		return formatterDocument.getSharedSettings().getFormattingSettings().getEnforceQuoteStyle();
 	}
+
+	private int getTabSize() {
+		return formatterDocument.getSharedSettings().getFormattingSettings().getTabSize();
+	}
+
+	private int getMaxLineWidth() {
+		return formatterDocument.getMaxLineWidth();

```

Comment:
@JessicaJHee I think we should give the capability to igonre the maxLineWidth settings (to format without max line width). To do that there are 2 options : 

 * keep getMaxLineWidth as int and check if getMaxLineWidth is set a given value like -1 or 0 which means that getMaxLineWidth format setting should be ignored.
 * replace getMaxLineWidth with Integer to have the capability to set a value to null

Reply:
I have made the changes so that if the maxLineWidth is set to 0, it is ignored.

Reply:
Thanks!

But please wrap this check !=0 in a New method like
formatterDocument.isMaxLineWidthSupported()